cdap-api/src/main/java/co/cask/cdap/api/Admin.java

```diff
@@ -17,80 +17,14 @@
 package co.cask.cdap.api;
 
 import co.cask.cdap.api.annotation.Beta;
-import co.cask.cdap.api.dataset.DatasetManagementException;
-import co.cask.cdap.api.dataset.DatasetProperties;
-import co.cask.cdap.api.dataset.InstanceConflictException;
-import co.cask.cdap.api.dataset.InstanceNotFoundException;
+import co.cask.cdap.api.dataset.DatasetManager;
 import co.cask.cdap.api.messaging.MessagingAdmin;
 import co.cask.cdap.api.security.store.SecureStoreManager;
 
 /**
  * This interface provides methods for operational calls from within a CDAP application.
  */
 @Beta
-public interface Admin extends SecureStoreManager, MessagingAdmin {
-  /**
-   * Check whether a dataset exists in the current namespace.
-   * @param name the name of the dataset
-   * @return whether a dataset of that name exists
-   * @throws DatasetManagementException for any issues encountered in the dataset system
-   */
-  boolean datasetExists(String name) throws DatasetManagementException;
+public interface Admin extends DatasetManager, SecureStoreManager, MessagingAdmin {
```

Comment:
unfortunately there is already a `DatasetAdmin` in api, which is for administration of single dataset type.